src/docs/asciidoc/wan/index.adoc

```diff
@@ -1,50 +1,76 @@
-in the New York cluster are to be synchronized to the Tokyo and London clusters.
+Assume that you have data centers in different cities each running an independent
+Hazelcast IMDG cluster. You can reliably use the WAN Replication feature
+to synchronize all of these clusters by replicating the updates to each of them.
 
-WAN Replication also is able to provide much more control than the replication mechanism between members in a single
-cluster with features and capabilities such as:
+WAN Replication provides more control compared to the replication mechanism between
+the members in a single cluster. It has the following features and capabilities:
 
-* WAN Replication will gracefully detect if there is a connectivity issue between clusters, buffering any updates
-that are not yet replicated and will attempt to re-establish a connection to resume replication
-* WAN Replication allows you to permanently pause, stop and resume replication. This is most useful when you know
-that one of the clusters will be temporarily (e.g. upgrade) or permanently (removing a cluster out of service) unavailable.
-* WAN Replication allows you to synchronize the contents of maps and caches
-* WAN Replication allows you to dynamically add new target clusters without any restarts
-And much more.
+* It gracefully detects if there is a connectivity issue between the clusters, buffering any updates
+that are not yet replicated and attempts to re-establish a connection to resume the replication.
+* It allows you to permanently pause, stop and resume the replication. This is most useful when you know
+that one of the clusters is temporarily (e.g., due to an upgrade),
+or permanently (e.g., due to removing a cluster out of service) unavailable.
+* It allows you to synchronize the contents of maps and caches.
+* It allows you to dynamically add new target clusters without any restarts.
 
-This section explains how you can replicate the state of your clusters over Wide Area Network (WAN)
-through Hazelcast WAN Replication.
+This chapter explains how you can replicate the state of your clusters over wide area
+networks through Hazelcast's WAN Replication.
 
-NOTE: You can download the white paper **Hazelcast on AWS: Best Practices for Deployment** at
-link:https://hazelcast.com/resources/amazon-ec2-deployment-guide/[Hazelcast.com^].
+NOTE: WAN Replication is a Hazelcast IMDG Enterprise Edition feature. However,
+its API is available publicly link:https://docs.hazelcast.org/docs/latest/javadoc/com/hazelcast/wan/package-summary.html[here^] and,
+benefiting from it, you may write your own replication logic.
 
-NOTE: To be able to use the REST calls related to WAN Replication mentioned in the following sections,
-you need to enable the `WAN` REST endpoint group.
-See the <<using-the-rest-endpoint-groups, Using the REST Endpoint Groups section>> for details.
+[[wr-concepts]]
+==== Concepts
 
+Let's first define several important terms before we discuss WAN Replication:
```

Comment:
I'm actually thinking if all of the concepts (or any actually) should be here. I would love to have the introduction followed directly with the quickstart. In my experience, I always first try copy pasting the code snippets and then trying to understand what it's doing. 
How would you feel about moving this section under "Defining WAN replication"?

Reply:
I am not sure about this one. When I read some instructions/guides in a document and want to find the related conceptual terms, I would mostly go to the intro part of that section, or to the glossary. Let me see what I can do; an option is distributing those concepts to their respective first mentions.
But still, it feels right to give them in the beginning.

Reply:
I would keep them together, as a reference. I don't mind whether they are at the beginning or the end. I prefer the beginning since we start using them extensively quite quickly and I wanted to avoid talking about "publisher" without first saying what it means. But maybe we can check out some other product documentation to see how others do it, maybe moving it to the end and adding a link from the intro 🤔